browser/ui/android/strings/android_brave_strings.grd

```diff
@@ -3891,6 +3890,18 @@ If you don't accept this request, VPN will not reconnect and your internet conne
       zip
     </message>
     <!-- End of Autofill Service strings -->
+    <message name="IDS_I_AGREE_ACTION" desc="CTA text for tos rewards panel">
+      I agree
+    </message>
+    <message name="IDS_REWARDS_TOS_TITLE" desc="Terms of service text for rewards panel">
+      Updated Terms of Service
+    </message>
+    <message name="IDS_REWARDS_TOS_UPDATE_TEXT" desc="Terms of service text for rewards panel">
+    We’ve updated the Terms of Service for Brave Rewards. We’ve made these updates to clarify our terms and ensure they cover new features. If you continue to use Brave Rewards, you are agreeing to the updated Terms of Service. If you prefer not to agree, you have the option to <ph name="RESET_START">&lt;reset&gt;</ph>reset<ph name="RESET_END">&lt;/reset&gt;</ph> Brave Rewards, and this will disable the feature.
```

Comment:
@deeppandya, the Desktop version is a little different. The last sentence on Desktop is: `If you do not agree, you can <ph/>reset<ph/> Brave Rewards, which will disable the feature.`. Are they intentionally different? If not, can they also be the same?

Reply:
i have used the text from : https://www.figma.com/file/8KhY2EucP83N0NqYspTImh/Terms-of-Service-update?type=design&node-id=3-39476&mode=design&t=1xwrEITDq2ygHCya-4. @Miyayes should we keep the text same ?

Reply:
Thanks @deeppandya, I'll approve but let's see what @Miyayes says.